src/robot/SessionManager.test.ts

```diff
@@ -26,7 +26,7 @@ const mockHealthyHeartbeat = (_req, _md, cb) => {
 describe('SessionManager', () => {
   beforeEach(() => {
     sm = new SessionManager(host, transport);
-    vi.mock('./gen/robot/v1/robot_pb_service');
+    vi.doMock('./gen/robot/v1/robot_pb_service');
```

Comment:
This path doesn't match the import above.

```ts
import { RobotServiceClient } from '../gen/robot/v1/robot_pb_service';
// ...
vi.doMock('./gen/robot/v1/robot_pb_service');
```

Is this mock doing anything? Commenting the mock out does not change the test outcome in any way that I can see

Reply:
Indeed - we stub out the relevant methods accessed in each test, so there's no need to mock the whole module.